src/cmd/describegpt.rs

```diff
@@ -0,0 +1,118 @@
+static USAGE: &str = r#"
+Infers extended metadata about a CSV using summary statistics.
+
+Note that this command uses a LLM for inference and is therefore prone to inaccurate
+information being produced. Ensure verification of output results before using them.
+
+For examples, see https://github.com/jqnatividad/qsv/blob/master/tests/test_describegpt.rs.
+
+Usage:
+    qsv describegpt [options] [<input>]
+    qsv describegpt --help
+
+describegpt options:
```

Comment:
we should also add a `--json` and `--pretty-json` output option, so the output is readily parseable by programs downstream.

Otherwise, the output will just be the generated text to stdout.

Reply:
Added `--json` and --pretty-json` output options.